ballerinaByExample/tools/generate.go

```diff
@@ -727,18 +727,18 @@ func main() {
         panic(err)
     }
 
-    er := os.Mkdir(siteDir + "/images", 0755)
-    if er != nil {
-        panic(er)
-    }
+    // er := os.Mkdir(siteDir + "/images", 0755)
```

Comment:
Shall we remove the line than comment out?

Reply:
Done.